metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java

```diff
@@ -33,8 +36,7 @@
     private WireMockServer wireMockServer;
 
     @Test
-    public void getMetadataReturnsSelectedMetadataFromURI()
-            throws ExtractionException, IOException, URISyntaxException {
+    public void getMetadataJsonReturnsCreatePublicationRequest() throws ExtractionException, IOException, URISyntaxException {
         String filename = TEST_REVIEW_PAPER_HTML;
         startMock(filename);
         var uriString = String.format(URI_TEMPLATE, wireMockServer.port(), filename);
```

Comment:
I think the lines until (including) the creation of json, belong to a method with name "createPublicationGraph" or something similiar